src/06-products/products.apib

```diff
@@ -41,3 +41,22 @@ Get details for a single product.
             + description: `dark chocolate` (string, nullable)
             + code: `COOK-DARKCHOC-42` (string, nullable)
             + purchase_price (Money, nullable, optional) - The purchase price will only be returned if you have access to it
+
+### products.add [POST /products.add]
+
+ Add a new product.
+
+ + Request (application/json;charset=utf-8)
+
+     + Attributes (object)
+        + name: `cookies` (string, optional) - You need to send name or code or both
```

Comment:
Wouldn't `you should send at least a name or a code` be a bit more clear language wise?